examples/chef/linux/main.cpp

```diff
@@ -49,6 +62,11 @@ int main(int argc, char * argv[])
     cmd_app_server_init();
 #endif
 
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+    // Send notification right away after init
+    DeviceLayer::SystemLayer().StartTimer(chip::System::Clock::Milliseconds32(0), notifyIcdActive, nullptr);
+#endif // CHIP_CONFIG_ENABLE_ICD_CIP
```

Comment:
This does not belong in main.cpp. Please create a new LIT ICD management file under chef/linux/